resources/views/components/cd/form/checkboxes.blade.php

```diff
@@ -0,0 +1,24 @@
+<x-cd.form.form-item field="{{ $name ?? $attributes->whereStartsWith('wire:model')->first() }}" 
+        classes="{{ $classes ?? '' }}" 
+        required="{{ (($required??'') === 'false') ? 0 : (boolval($required??'')?1:0) }}"
+        :description="$description ?? ''"
+>
+@php 
+  $field=$name ?? $attributes->whereStartsWith('wire:model')->first();
+@endphp
+    <x-slot name="field_title">{!! $label !!}</x-slot>
+    <div class="flex-grid compact-rows">
```

Comment:
What would you think of putting $slot between these:
```suggestion
    <x-slot name="field_title">{!! $label !!}</x-slot>
    {!! $slot !!}
    <div class="flex-grid compact-rows">
```
That would make it possible to include content between the heading and the options, like in the ["A Plethora of Checkboxes"](https://iws-preview.hosting.cornell.edu/ama39/cssf/forms.html#section-12) example